### ISSUE TITLE
NullReferenceException

### ISSUE THREAD

**Issue by Allann** [object/Simple.OData.Client#72]
If we add the attribute [Column(Typename="varchar")] to a property so that EF6 makes the property a varchar field and not an nvarchar field, OData will fail in MemberInfoExtensions.GetMappedName

A simple change to the last check in the function will fix this.

Old:
```c#
            if (nameProperty != null) propertyName = nameProperty.GetValue(mappingAttribute, null).ToString();
```

New:
```c#
            if (nameProperty != null)
            {
                var prop = nameProperty.GetValue(mappingAttribute, null);
                if(prop!=null) return prop.ToString();
            }
```

**Comment by object** [comment]
Thanks, I'll update asap.

**Comment by object** [comment]
Fixed in 4.0.9.
